src/map/ai/ai_mob_dummy.cpp

```diff
@@ -408,18 +408,18 @@ void CAIMobDummy::ActionDropItems()
 				blueutils::TryLearningSpells(PChar, m_PMob);
 				m_PMob->m_UsedSkillIds.clear();
 
-				if(m_PMob->m_giveExp)
-				{
-	                charutils::DistributeExperiencePoints(PChar, m_PMob);
-				}
+                if(m_PMob->m_giveExp && m_PMob->getMobMod(MOBMOD_EXP_BONUS) >= 0)

```

Comment:
yeah.. i'm pretty sure this will make every mob that doesn't have this mod give 0 exp